File: trunk/proj/CoreLibrary/src/com/jasonzqshen/familyaccounting/core/exception/DocReservedException.java
package com.jasonzqshen.familyaccounting.core.exception;

public class DocReservedException extends Exception {
	/**
	 * 
	 */
	private static final long serialVersionUID = -2168800139866243287L;

	public DocReservedException() {
		super("Document has been reserved before.");
	}
}

File: trunk/proj/WinStore_FamilyAccounting/WinStore_FamilyAccountingCore/Exceptions/DocReservedException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WinStore_FamilyAccountingCore.Exceptions
{
    public class DocReservedException : Exception
    {
        public DocReservedException() : base("Document has been reserved before.")
        {
        }
    }
}
